app/public/covid.css

```diff
@@ -71,6 +71,11 @@ div.doubleblock {
 .footer-text {
     text-align: center;
 }
+<<<<<<< HEAD
```

Comment:
Can you please solve this conflict, as this is breaking the responsive behaviour of the .center class

Reply:
Added missing closing bracket. Fixed